lametro/forms.py

```diff
@@ -135,4 +135,68 @@ def clean_agenda(self):
           if isinstance(agenda_pdf, InMemoryUploadedFile) and agenda_pdf.name.lower().endswith('pdf'):
               return agenda_pdf
           else:
-              raise forms.ValidationError('File type not supported. Please submit a PDF.')
\ No newline at end of file
+              raise forms.ValidationError('File type not supported. Please submit a PDF.')
+
+
+class PersonHeadshotForm(forms.ModelForm):
```

Comment:
Ah, I see how could interpret what I asked for to be the image URL directly, and this achieves that! A better UX would be to upload a file, have the app store that file in the configured storage backend, and save the URL to the file in our backend.

We have a pattern for storing user uploads directly on the server, but looking ahead, it would be more future proof to store user uploads in S3 so the app is portable. `django-storages` provides an easy interface for working with S3. 

What do you think about updating this form to accept a file, then uploading the file and storing the URL on the model in the `form_valid` method of the view? 

See [the docs](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html) (specifically Usage for configuration, then down to "Or you may want to upload files to the bucket in some view that accepts file upload request...") for an example of how to upload to S3!

Reply:
Oh I getcha. This sounds good! In that case, I'm looking through the docs to get django-storages set up, and it looks like I'll probably need an account/bucket made for aws keys, is that right? Do we have anything I should use, or should I just make one?

Reply:
I hit a snag here, what do you think about this: 

I was trying to add a FileField for headshots to LAMetroPerson so I could get them into the bucket I made for this, but I found that you can’t add a field to a proxy model. I asked in chat and got a couple suggestions to potentially either 
- add this field onto django-councilmatic's Person model, or
- create a different model for the file field and refer to it via foreign key on the LAMetroPerson model

I figured the second option might be better, but Derek rightfully suggested that I consult with you after the break before going down either route.